app/views/paye/paye_tax_free_amount.scala.html

```diff
@@ -52,12 +52,14 @@ <h1 class="heading-xlarge">
 		</tr>
 		}
 		</tbody>
-		<tfoot>
-		<tr id="totalTaxFreeAmount">
-			<th scope="row">@Messages("paye.ats.tax_free_amount.total_tax_free_amount")</th>
-			<td class="numeric">@includes.currency_amount(viewModel.totalTaxFreeAmount, true)</td>
-		</tr>
-		</tfoot>
+		@if(viewModel.adjustmentRows.size > 1) {
```

Comment:
Is there a test for viewModel.adjustmentRows.size == 1?

Reply:
I've added that test below for when viewModel.adjustmentRows.size == 1 and to still display the table without a total row. Do you think another one is required?